controller-server/src/test/java/com/yahoo/vespa/hosted/controller/restapi/application/ApplicationApiTest.java

```diff
@@ -1646,6 +1647,7 @@ public void testJobStatusReporting() {
                 .projectId(projectId);
         job.type(JobType.component).uploadArtifact(applicationPackage).submit();
         controllerTester.deploy(app.id().defaultInstance(), applicationPackage, TEST_ZONE);
+        ((ManualClock) controllerTester.controller().clock()).advance(Duration.ofSeconds(1));
```

Comment:
Nit: If you access it through `controllerTester.serviceRegistry()` you can get rid of the cast.

Reply:
Hehe, yes.